docs/request.md

```diff
@@ -82,11 +83,11 @@ The `m.request` utility is a thin wrapper around [`XMLHttpRequest`](https://deve
 
 ```javascript
 m.request({
-	method: "GET",
-	url: "/api/v1/users",
+  method: "GET",
+  url: "/api/v1/users",
```

Comment:
Why are whitespace changes included in this PR?

Reply:
My bad. I use a markdown linter, and it replaced them all...